Wikipedia/Code/ArticleViewController+Editing.swift

```diff
@@ -184,7 +184,7 @@ extension ArticleViewController: SectionEditorViewControllerDelegate {
         case .success(let changes):
             dismiss(animated: true)
             waitForNewContentAndRefresh(changes.newRevisionID)
-            EditAttemptFunnel.shared.logSaveSuccess(articleURL: self.articleURL)
+            EditAttemptFunnel.shared.logSaveSuccess(articleURL: self.articleURL, revisionId: Int(changes.newRevisionID))
```

Comment:
No change needed here, and it looks like we already use this pattern elsewhere, but just wanted to mention: generally for important casting situations where there's possible precision loss, I'll just future proof and use the higher level `NSNumber` and allow it to handle doing the right thing, so something like `NSNumber(value: changes.newRevisionID).intValue` here. But my understanding of this specific case is that we're ok without that, with `Int` on the devices that can run our app itself being a `Int64` and it being unlikely we hit the max value revision id.